src/extension/tab.js

```diff
@@ -61,9 +62,6 @@ async function getProxyUrl({ id, url: tabUrl }) {
  * @param {Object[]} matches The config matches
  */
 async function injectContentScript(tabId, matches) {
-  if (!await chrome.tabs.get(tabId)) {
```

Comment:
I don't think it's possible to hit this code path now

Reply:
I'm not sure... this was an attempt to prevent the "Uncaught (in promise) Error: No tab with id: 12345678" errors we are getting super often in the extension's error log...